src/cloudant/database.py

```diff
@@ -919,7 +919,7 @@ def delete_index(self, design_document_id, index_type, index_name):
             raise CloudantArgumentError(msg)
         index.delete()
 
-    def get_query_result(self, selector, fields=None, raw_result=False, **kwargs):
+    def get_query_result(self, selector, raw_result=False, **kwargs):
```

Comment:
I would also add 2 tests to the `database_tests.py` that ensure that when passing either None or an empty list to `fields` returns all of the fields expected.